bin/aks-create-cluster.sh

```diff
@@ -25,6 +25,8 @@ echo -e "\tNode VM Size = ${AKS_NODE_VM_SIZE}"
 echo -e "\tNode Disk Size = ${AKS_NODE_OSDISK_SIZE}"
 echo -e "\tDefault Namespace = ${AKS_CLUSTER_NS}"
 echo -e "\tMonitoring Namespace = ${AKS_MONITORING_NS}"
+echo -e "\tStatic IP = ${AKS_INGRESS_IP}"
+echo -e "\tStatic IP Resource Group = ${AKS_IP_RESOURCE_GROUP_NAME}"
```

Comment:
echo -e "\tIngress Controller IP = ${GKE_INGRESS_IP}"

Wajih used this string in gke-create-cluster.sh. Either is OK, but could you make them consistent (either change this or change gke-create-cluster.sh) for a more polished look and feel?

Reply:
ah yes, missed that.

Reply:
done